docs/getting-started/tutorial-sample-query.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[tutorial-sample-query]]
+=== Querying the data
+
+You can also enter an {es} query to narrow the view of the data.  {kib} supports the 
+<<lucene-query>>.
+
+. In the query bar, enter this simple query:
++
+[source,text]
+OriginCityName:Rome   
++
+The dashboard now shows data for flights out of Rome. 
+
+. For a more complex query with AND and OR, try this:
++
+[source,text]
+OriginCityName:Rome AND (Carrier:JetBeats OR "Kibana Airlines")
++
+The dashboard updates to show data for the flights out of Rome on JetBeats and 
+{kib} Airlines.  
++
+[role="screenshot"]
```

Comment:
This seems to be the screenshot for the input controls still. Or at least it is on the hosted page.